src/index.js

```diff
@@ -82,13 +112,9 @@ class FloodSub extends EventEmitter {
 
   _onDial (peerInfo, conn, callback) {
     const idB58Str = peerInfo.id.toB58String()
+    log('connected', idB58Str)
 
-    // If already had a dial to me, just add the conn
-    if (!this.peers.has(idB58Str)) {
-      this.peers.set(idB58Str, new Peer(peerInfo))
-    }
```

Comment:
This is important logic.

What is happening here is: "If the other peer has already dialed to me, we already have an establish link between the two, what might be missing is a Connection specifically between me and that Peer"

Reply:
This was copy and paste from original code.  will add the comment.

Reply:
The code was moved to `_addPeer`